Update connecting-a-code-repository.md (#1878)
spryker/spryker-docs: docs/cloud/dev/spryker-cloud-commerce-os/connecting-a-code-repository.md

{% info_block infoBox "Initial setup" %}

If you specified a code repository in the Onboarding Questionnaire, we connected it to your Staging environment during the initial setup. To connect a code repository to another environment, follow this document. 

> Please note that we are only supporting the Cloud variants of the repositories listed below. If you need to connect a On Premises repository, please reach out to our support to discuss potential options.

Reviewer: ```suggestion

Note that we are only supporting the Cloud variants of the repositories listed below. To connect the On-Premises repository, reach out to our [support team](https://spryker.force.com/support/s/) to discuss potential options.
```
